packages/angular_devkit/core/src/_golden-api.ts

```diff
@@ -9,7 +9,7 @@
 export * from './exception/exception';
 
  // Start experimental namespace
-export * from './workspace/index';
+export * from './experimental/workspace/index';
```

Comment:
The existing workspace elements were already experimental from the public API surface.  The file structure was misaligned in that aspect.  The first commit in the PR corrects this and moves the actual files into a more appropriate location.  These existing elements are still available as the new subsystem is not yet exposed publicly as this PR represents the core infrastructure for additional work which will be completed in subsequent PRs.

Reply:
I agree that the old one was in the wrong folder and that `refactor(@angular-devkit/core): move existing workspace API under experimental` corrects that.

But I don't agree with keeping the old one and new one side by side in master. It feels to me like it would cause unnecessary extra work to develop it master. Mostly extra reviews, API updates, and rot prevention. 

The alternative would be to develop this replacement in a feature branch and wholly replace the old workspace when it is ready. This seems like a better alternative to me.

Are there reasons why the feature branch option is inadvisable here?

Reply:
Discussed offline, this phased development model had already been discussed but I was not aware.